ultratrace2/app.py

```diff
@@ -15,10 +18,41 @@ def __init__(self, args):  # FIXME: be more granular here
         else:
             path = args.path
 
-        self.project = Project(path)
+        # FIXME: allow changing the $HOME-as-root via command line arg?
+        self.ultratrace_home = os.path.join(os.environ["HOME"], ".ultratrace")
```

Comment:
From what I can tell, Windows might not have this set. I think this does what you want on any platform
```suggestion
        self.ultratrace_home = os.path.expanduser(os.path.join("~", ".ultratrace"))
```

Reply:
I'm unclear on whether we intend to support Windows, @jonorthwash is this a goal of ours?

Reply:
In any case, 7c46ccb should supercede this

Reply:
Yeah, in theory we'd like to support Windows.  But I still haven't fully tested on Windows (I did a little bit yesterday, but couldn't quite get it working—there was an issue with `libmagic` or something.)

Reply:
Is the goal here to store app preferences in the user's home directory?

Reply:
So I think I'm abandoning storing anything in the home directory, since I don't think we'll need that. Maybe in the future it would be nice to store preferences (probably in `$XDG-*` if available), but I don't have a use-case yet.

Regarding `libmagic`, I think this is also obsolete as of [this commit](https://github.com/SwatPhonLab/UltraTrace/commit/04c8af2a13e61f0b80a46755ce1696a86288d81c), which removes the `python-magic` dependency in favor of the standard `mimetypes` lib.